site/theme/en-US.js

```diff
@@ -11,5 +11,33 @@ module.exports = {
     'app.header.menu.spec': 'Specification',
     'app.header.menu.resource': 'Resource',
     'app.header.lang': '中文',
+    'app.home.slogan': 'A design language for UI',
+    'app.home.start': 'Get Started',
+    'app.home.best-practice': 'Best Practice',
+    'app.home.experience': 'Designed by experienced team, and showcase dozens of inspiring projects.',
+    'app.home.design-pattern': 'Design Pattern',
+    'app.home.pattern': 'Provide solution for usual problems which we may meet in developing enterprise-like complex UI.',
+    'app.home.reusable-components': 'Dozens of Reusable Components',
+    'app.home.components-intro': 'Dozens of flexible and practical reusable components, which will power productivity up.',
```

Comment:
'app.home.components-intro': 'Dozens of flexible and practical reusable components that increase your productivity.',